services/common/public_key_url_allowlist.h

```diff
@@ -32,6 +32,8 @@ inline constexpr absl::string_view kGCPProdPublicKeyEndpoint =
     "https://publickeyservice.pa.gcp.privacysandboxservices.com/.well-known/protected-auction/v1/public-keys";  // NOLINT(whitespace/line_length)
 inline constexpr absl::string_view kAWSProdPublicKeyEndpoint =
     "https://publickeyservice.pa.aws.privacysandboxservices.com/.well-known/protected-auction/v1/public-keys";  // NOLINT(whitespace/line_length)
+inline constexpr absl::string_view kAzureProdPublicKeyEndpoint =
```

Comment:
Not sure if we can claim this URL.

Reply:
Yeah, agree. Do you have any specific idea for URL? We could put some dummy URL and put comment to mention about it as well

Reply:
Let's check with Brandon.

Reply:
Fixed in [6638f72](https://github.com/KenGordon/bidding-auction-servers/pull/26/commits/6638f72054fa8a5b7a7fbbdedfd662ff54c05f41)